services/ui-src/src/components/pages/ReviewSubmit/McparReviewSubmitPage.test.tsx

```diff
@@ -66,11 +60,12 @@ describe("Test McparReviewSubmitPage functionality", () => {
     expect(screen.getByText(intro.infoHeader)).toBeVisible();
   });
 
-  it("should navigate to the print preview page on button click", async () => {
+  it("print button should be visible and correctly formed", async () => {
     render(McparReviewSubmitPage_InProgress);
     const printButton = screen.getByText("Print");
-    await userEvent.click(printButton);
-    expect(mockUseNavigate).toHaveBeenCalledWith("/mcpar/export");
+    expect(printButton).toBeVisible();
+    expect(printButton.getAttribute("href")).toEqual("/mcpar/export");
+    expect(printButton.getAttribute("target")).toEqual("_blank");
   });
```

Comment:
Author note: I modified both of these tests because we aren't using `useNavigate` anymore. Now that we're just using a `<Button as={RouterLink} />` there's no real implementation that I care about testing, so I'm just testing that we're giving it the correct attributes to get the desired outcome.